### DISCUSSION TITLE
Login Screen issue.

### DISCUSSION THREAD

**Discussion by klajdi-g** [hyprwm/Hyprland#5785] [General]
Hello, I am having an error with the Lock Screen UI.
It keeps giving this message even tho I manually installed the dependencies
"The current theme cannot be loaded due to the errors below, please select another theme.
files/uar/share/sddm/themes/simple-sddm-2/Main.qml: Library import requires a version"

**Comment by Agent00Ming** [comment]
Coming from KDE Plasma? (likely) You're going to have to find and replace the themes placed there by plasma settings with your own SDDM themes or some you find on the web.

**Reply by Agent00Ming** [reply]
Otherwise, the theme needs some tweaking. Might be worth looking at what the qml file does

**Reply by klajdi-g** [reply]
Ok so first time i booted up it gave me this
![Uploading IMG_3313.jpeg…]() [Image isn't loading but it just said (Library import requires a version)]
I tried to fix an imported library by just adding a version to it(i thought it would work) and when i did and restarted my laptop it gave me this 
![IMG_3314](https://github.com/hyprwm/Hyprland/assets/115544942/e8b14746-ffbb-47d3-8024-71d79ff38fd4)
Now when im trying to download the package my terminal wont even find it.

**Reply by klajdi-g** [reply]
Also, when i log in into the Hyperland "Session" everything seems to be working completely fine? just the lockscreen is glitched out.

**Reply by Agent00Ming** [reply]
Yes, it's a theme issue. Undo the version setting and install QT5 package group if you want to use this theme

**Reply by klajdi-g** [reply]
Hello, yes i did undo the version setting and installed QT5 package group,I restarted my laptop to see if the login screen theme is fixed, but it wasn't.
Maybe I should delete it completely?
I also tried to just download the package and it downloaded, but when I restarted the laptop the login screen just gave the same error saying GrahpicalEffects is not installed.

**Reply by Agent00Ming** [reply]
Maybe try the QT6 package group to see if it has the required dependency then move on to another theme or something else

**Reply by klajdi-g** [reply]
Yes i tried to fix it by downloading the QT6 package group but it came into conflict with QT5 and now when I turn on my laptop there's no GUI,only a cli to log in through lol.

**Reply by Agent00Ming** [reply]
You might be better off uninstalling any sddm themes that are installed and look for one that is more updated

**Reply by klajdi-g** [reply]
yeah i just deleted the theme completely and I installed a cattppucinn theme and it worked thank you <3.
I was kinda worried cuz its my first time using Hyprland lol.
